.eslintrc

```diff
@@ -1,6 +1,7 @@
 {
 	"env": {
-		"node": true
+		"node": true,
+    "es6": true
 	},
 	"plugins": [
 		"nodeca"
```

Comment:
Remove this plugin and replace with the eslint-plugin-node, then check webpack/webpack at copy their eslintrc file as well as copy the `engine` property from webpack/webpack's package.json file and the same value. Otherwise looks great!!